deploy/rand.go

```diff
@@ -0,0 +1,44 @@
+package deploy
+
+import (
+	crand "crypto/rand"
+	"encoding/binary"
+	mrand "math/rand"
+)
+
+var source = mrand.New(&cryptoSource{})
```

Comment:
You certainly don't need this file at all to get a single random `uint32`.

Reply:
Replaced with `rand.Uint32()`